app/code/Magento/Quote/Model/Quote/Address/Total.php

```diff
@@ -166,8 +173,7 @@ public function getAllBaseTotalAmounts()
     //@codeCoverageIgnoreEnd
 
     /**
-     * Set the full info, which is used to capture tax related information.
-     * If a string is used, it is assumed to be serialized.
+     * Set the full info, which is used to capture tax related information. If a string is used, it is assumed to be serialized.
```

Comment:
Please re-split the comment into two lines

Reply:
An error of "short description must not be multilined" shows up in Travis then

Reply:
The only failed test is because of the short description